go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -537,6 +540,32 @@ func objName(parts ...string) string {
 	return res
 }
 
+// this is a helper to calculate the part size, taking into consideration the minimum part size
+// passed in by an operator.
+func getPartSize(filesize int64) (partSizeBytes int64, err error) {
+	// Calculate s3 upload part size using the source filesize
+	partSizeBytes = manager.DefaultUploadPartSize
+	if filesize > 0 {
+		minimumPartSize := float64(filesize) / float64(manager.MaxUploadParts)
+		// Round up to ensure large enough partsize
+		calculatedPartSizeBytes := int64(math.Ceil(minimumPartSize))
+		if calculatedPartSizeBytes > partSizeBytes {
+			partSizeBytes = calculatedPartSizeBytes
+		}
+	}
+
+	if minimumPartSize != 0 && partSizeBytes < minimumPartSize {
+		if minimumPartSize > 1024*1024*1024*5 || minimumPartSize < 1024*1024*5 { // 5GiB and 5MiB respectively
```

Comment:
Where do these min and max values come from? I would hope that there are consts we can use instead?

Reply:
that's a great point. for the minimum, the SDK does define it, so we could use `manager.MinUploadPartSize`, but I don't see anything for the max. I will take a better look when I update the code for the flag change mentioned above, but we can at least change that one